.github/workflows/test.yml

```diff
@@ -72,3 +72,16 @@ jobs:
         run: |
           cargo r -- cargo-quickinstall
           cargo quickinstall -V
+
+      - name: Test batch installation with binstall
+        run: |
+          cargo r -- cargo-quickinstall cargo-nextest
```

Comment:
this should fail on windows.... except it doesn't because we `exec` to run binstall, so we're no longer running? or does it noop because it's already installed?

Reply:
> this should fail on windows....

Why shall this fail though?

> except it doesn't because we exec to run binstall

We only do exec on unix.

> or does it noop because it's already installed?

Yes, `cargo-binstall` checks local manifest and do nothing if it's already installed.